docker/development.yml

```diff
@@ -59,3 +59,10 @@ services:
   elasticsearch:
     ports:
       - '9200:9200'
+
+  # Users
+  users:
+    ports:
+      - ${USERS_PORT}:${USERS_PORT}
```

Comment:
Why are you exposing this on the host in development?

Reply:
So the docker containers can talk to each other when running in dev mode with Docker